s04e03/episode.txt

```diff
@@ -12,4 +12,74 @@ ADDITIONAL:
 RATING: Explicit
 YOUTUBE: hYk1_V4-DcQ
 EDITOR: Tele (Tele42)
-NOTESCREATOR:
+NOTESCREATOR: nixi
+
+00:00:07 *Introduction*
+00:00:16 HeroOhOne begins the introduction of co-hosts
+00:00:19 @hmccio
+00:00:20 DerRidda
+
+*Our Gaming*
+00:00:27 @hmccio has been playing Slime Rancher http://www.slimerancher.com/
+00:01:17 @HeroOhOne has been playing Natural Selection 2 http://www.naturalselection2.com/, Trine 3 http://trine3.com/, GRID: Autosport http://www.gridgame.com
+00:02:17 @DerRidda has been playing Insurgency https://newworldinteractive.com/#insurgency, Skyborn http://store.steampowered.com/app/278460/, XCOM2 https://xcom.com
+
+*SteamLUG Community Stuff*
+00:02:55 Past Events
+	Starbound http://steamcommunity.com/groups/steamlug#events/92687321861449828
+	7 Days to Die http://steamcommunity.com/groups/steamlug#events/92687321861457895
+	PULSAR: Lost Colony http://steamcommunity.com/groups/steamlug#events/92687321861462229
+	Saints Row http://steamcommunity.com/groups/steamlug#events/92687321861472204
+00:04:52 Upcomming Events
+	Counter-Strike: Global Offensive http://steamcommunity.com/groups/steamlug#events/92690048508262847
+	DiRT Showdown http://steamcommunity.com/groups/steamlug#events/92690048508263024
+	Insurgency http://steamcommunity.com/groups/steamlug#events/92690048508270992
+	Guns of Icarus Online http://steamcommunity.com/groups/steamlug#events/92690048508271209
+00:05:50 @HeroOhOne got his Thrustmaster TX xbone wheel initialized in Linux https://github.com/her001/tmdrv
+
+*Steam/Valve News*
+00:07:16 Steam bug which did not allow Chromium-based games or applications to be launched from the Linux client has been fixed http://steamcommunity.com/groups/SteamClientBeta#announcements/detail/903337784334862903
+00:08:01 Lunar New Year Sale http://store.steampowered.com/news/20313/
+
+*Tech News*
+00:09:15 AMD GPUOpen launched http://gpuopen.com/welcometogpuopen/
+	At least 30 git repos spread over 4 github organizations https://github.com/GPUOpen-LibrariesAndSDKs, https://github.com/GPUOpen-Effects, https://github.com/GPUOpen-Tools, https://github.com/GPUOpen-ProfessionalCompute-Tools
```

Comment:
I was not able to find a different shownotes file with more than one link on the same line. 
Will this work?